Fix buld failure of USE=-ipv6 case.

> netaddr.c: In function 'get_addr_by_name':
> netaddr.c:694:3: warning: control reaches end of non-void function [-Wreturn-type]
>    }
>    ^

Fixes: http://bugs.gentoo.org/465134
Patch-by:  Yuri Shatroff <yks-uno@ya.ru>
diff --git a/src/netaddr.c b/src/netaddr.c
index 9dbc716..2b025b6 100644
--- a/src/netaddr.c
+++ b/src/netaddr.c
@@ -690,8 +690,8 @@ static pr_netaddr_t *get_addr_by_name(pool *p, const char *name,
         pr_freeaddrinfo(info);
       }
     }
-#endif /* PR_USE_IPV6 */
   }
+#endif /* PR_USE_IPV6 */
 
   return na;
 }
